packages/vuetify/src/util/helpers.ts

```diff
@@ -41,31 +56,20 @@ export function addOnceEventListener (
     el.removeEventListener(eventName, once, options)
   }
 
-  el.addEventListener(eventName, once, options)
+  addEventListener(el, eventName, once, options)
 }
 
-let passiveSupported = false
-try {
-  if (typeof window !== 'undefined') {
-    const testListenerOpts = Object.defineProperty({}, 'passive', {
-      get: () => {
-        passiveSupported = true
-      },
-    })
-
-    window.addEventListener('testListener', testListenerOpts, testListenerOpts)
-    window.removeEventListener('testListener', testListenerOpts, testListenerOpts)
-  }
-} catch (e) { console.warn(e) }
-export { passiveSupported }
-
 export function addPassiveEventListener (
```

Comment:
I don't understand why you're keeping this, it doesn't do anything unless you give an options object, then it ignores the options you give it.

Reply:
Well it did it's job incorrectly, I fixed it.